scripts/validate.py

```diff
@@ -191,7 +206,8 @@ def validate(schema_name, src_path):
         for item in items:
             try:
                 print("Validate schema for {}".format(item["id"]))
-                jsonschema.validate(instance=item, schema=schema)
+                schema_or_exception = load_schema_exceptions(item.get("template")) or schema
```

Comment:
This feels a little bit confusing to me. Now that we have a dedicated schema for Moments, how about we split them from the "cfr.yaml" to be a top-level asset, like `moments.yaml`? Then we can add a new line to the `check` section in the makefile.

Reply:
much better yes